src/test/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectSecurityAwarenessScoreTest.java

```diff
@@ -19,59 +20,148 @@
   @Test(expected = IllegalArgumentException.class)
   public void noInfoAboutSecurityAdvisories() {
     PROJECT_SECURITY_AWARENESS.calculate(HAS_SECURITY_TEAM.value(false));
+    PROJECT_SECURITY_AWARENESS.calculate(USES_VERIFIED_SIGNED_COMMITS.value(false));
   }
 
   @Test(expected = IllegalArgumentException.class)
   public void noInfoAboutSecurityTeam() {
     PROJECT_SECURITY_AWARENESS.calculate(HAS_SECURITY_POLICY.value(false));
+    PROJECT_SECURITY_AWARENESS.calculate(USES_VERIFIED_SIGNED_COMMITS.value(false));
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void noInfoAboutSignedCommits() {
+    PROJECT_SECURITY_AWARENESS.calculate(HAS_SECURITY_POLICY.value(false));
+    PROJECT_SECURITY_AWARENESS.calculate(HAS_SECURITY_TEAM.value(false));
   }
 
   @Test
   public void test() {
```

Comment:
Could you please remove the entire test case since it duplicates the test vectors?